x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/components/form_flyout.test.tsx

```diff
@@ -12,48 +12,81 @@ import {
 } from '../../../../../common/mock/endpoint';
 import userEvent from '@testing-library/user-event';
 import { HostIsolationExceptionsFormFlyout } from './form_flyout';
-import { act } from 'react-dom/test-utils';
 import { HOST_ISOLATION_EXCEPTIONS_PATH } from '../../../../../../common/constants';
 import uuid from 'uuid';
 import { createEmptyHostIsolationException } from '../../utils';
+import { waitFor, waitForElementToBeRemoved } from '@testing-library/react';
+import {
+  createHostIsolationExceptionItem,
+  updateOneHostIsolationExceptionItem,
+  getOneHostIsolationExceptionItem,
+} from '../../service';
 
 jest.mock('../../service.ts');
 jest.mock('../../../../../common/hooks/use_license');
 
+const createHostIsolationExceptionItemMock = createHostIsolationExceptionItem as jest.Mock;
+const updateOneHostIsolationExceptionItemMock = updateOneHostIsolationExceptionItem as jest.Mock;
+const getOneHostIsolationExceptionItemMock = getOneHostIsolationExceptionItem as jest.Mock;
+
 describe('When on the host isolation exceptions flyout form', () => {
   let mockedContext: AppContextTestRender;
   let render: () => ReturnType<AppContextTestRender['render']>;
   let renderResult: ReturnType<typeof render>;
-  let waitForAction: AppContextTestRender['middlewareSpy']['waitForAction'];
+  let onCancel: () => void;
 
-  beforeEach(() => {
+  beforeEach(async () => {
+    onCancel = jest.fn();
     mockedContext = createAppRootMockRenderer();
-    render = () => {
-      return mockedContext.render(<HostIsolationExceptionsFormFlyout />);
-    };
-    waitForAction = mockedContext.middlewareSpy.waitForAction;
+    mockedContext.history.push(`${HOST_ISOLATION_EXCEPTIONS_PATH}?show=create`);
```

Comment:
do we have a `routing` method to get paths for host isolation exceptions? If so, I would suggest using it here instead of hardcoding the path.